### PR TITLE
Config args

### PR DESCRIPTION
**Pull request by mnunberg**

### CODE REVIEW

File: src/config.c [hunk 1]
```diff
@@ -2,41 +2,31 @@
...
+#define RETURN_PARSE_ERROR(rc)                                    \
+  QueryError_SetError(status, QUERY_EPARSEARGS, AC_Strerror(rc)); \
+  return REDISMODULE_ERR;
 
-static int readLongLong(RedisModuleString **argv, size_t argc, size_t *offset, long long *out) {
-  assert(*offset <= argc);
-  if (*offset == argc) {
-    // printf("Missing argument!\n");
-    RETURN_ERROR("Missing argument");
+#define MAYBE_RETURN_PARSE_ERROR(rc) \
```
Comment (MeirShpilraien) [inline src/config.c]: I do not like the MAYBE, how about CHECK_AND_RETURN_PARSE_ERROR?

File: src/config.c [hunk 2]
```diff
@@ -113,12 +92,8 @@ CONFIG_GETTER(getForkGCSleep) {
 
 // MAXDOCTABLESIZE
 CONFIG_SETTER(setMaxDocTableSize) {
-  long long size;
-  if (readLongLongLimit(argv, argc, offset, &size, 1, MAX_DOC_TABLE_SIZE) != REDISMODULE_OK) {
```
Comment (MeirShpilraien) [inline src/config.c]: Where do you specify the max limitation?

File: src/config.c [hunk 3]
```diff
@@ -234,39 +188,23 @@ CONFIG_GETTER(getGcScanSize) {
...
 }
 
 CONFIG_SETTER(setForkGcCleanThreshold) {
-  long long val;
-  if (readLongLongLimit(argv, argc, offset, &val, 0, LLONG_MAX) != REDISMODULE_OK) {
-    return REDISMODULE_ERR;
-  }
-  config->forkGcCleanThreshold = val;
-  return REDISMODULE_OK;
+  int acrc = AC_GetSize(ac, &config->forkGcCleanThreshold, 0);
```
Comment (MeirShpilraien) [inline src/config.c]: Why not AC_F_GE0?

Comment (mnunberg) [reply src/config.c]: because size_t is an unsigned type, GE_0 is implicit

**Pull request merged**